src/components/products/DetailCard.jsx

```diff
@@ -1,15 +1,19 @@
 /* eslint-disable no-restricted-syntax */
 /* eslint-disable react/prop-types */
-import { useState } from "react";
-import useUserRoleState from "../../utils/isLogged";
-import Seller from "./utils/Seller";
-import AwesomeSlider from "react-awesome-slider";
+import { useState } from 'react';
+import useUserRoleState from '../../utils/isLogged';
+import Seller from './utils/Seller';
+import Buyer from './utils/Buyer';
+import AwesomeSlider from 'react-awesome-slider';
 import 'react-awesome-slider/dist/styles.css';
```

Comment:
Please let's separate built-in library imports and our own imports. take a look

we are importing {useState } from react  and the next lines you are importing {useUserRoleState , Seller & Buyer  } as your own fucntions then the next is {AwesomeSlider } from "react-awesome-slider".  if you see this, we are mixing things. **THE BEST IS:**

1. Built-in libraries imports must come first
2. Leave the space (one empty line) between built-in libraries and user-defined one
3. Again leave the space between the imports and the start of the Function/ Component definition (in this case "DetailCard ")
4. Let's use the Arrow function: it means from ` function DetailCard (props) {}` to 

```
const DetailCard = (props) => {
  // Function body
};
```